cli-plugins/manager/candidate_test.go

```diff
@@ -48,7 +49,7 @@ func TestValidateCandidate(t *testing.T) {
 		},
 	})
 
-	for _, tc := range []struct {
+	for i, tc := range []struct {
```

Comment:
It'd be really nice to give these test cases a name and use `t.Run`.
Debugging with an index like this is painful.

Reply:
Exactly what i was struggling with, hence the `i` that i wanted to remove.